lightning/src/onion_message/messenger.rs

```diff
@@ -41,7 +41,7 @@ use crate::prelude::*;
 ///
 /// # Example
 ///
-/// ```
+/// 
```

Comment:
Why remove these? This disables the doc tests.

Reply:
These ` ``` ` were giving error during cargo test.